packages/components/organisms/f-checkout/src/demo/customer-auth.json

```diff
@@ -0,0 +1,3 @@
+{
+    "token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImpvZS5ibG9nZ3NAanVzdGVhdHRha2Vhd2F5LmNvbSIsImNyZWF0ZWRfZGF0ZSI6IjIwMjEtMDItMDhUMTA6Mjc6NDkuMTkzMDAwMFoiLCJuYW1lIjoiSm9lIEJsb2dncyIsImdsb2JhbF91c2VyX2lkIjoiVTdOUkFsV0FnNXpPZHNkUmdmN25rVHlvaTkwWEVvPSIsImdpdmVuX25hbWUiOiJKb2UiLCJmYW1pbHlfbmFtZSI6IkJsb2dncyIsImlhdCI6MTYxNTQ2OTUxNn0.VapH6uHnn4lHIkvN_mS9A9IVVWL0YPNE39gDDD-l7SU"
```

Comment:
Not great for formatting as the line is too long but unfortunately a token with all the right information is that long!

Reply:
Do we need this file? We normally have these JSON files to mock APIs.

Reply:
Yeah it was something I was wondering. I wasn't exactly sure where to keep something like this which is only for Storybook and testing purposes

Reply:
Maybe in the `setup.js` file for the tests and then `Checkout.stories.js` for the story, I'd say.